go-controller/pkg/config/config_test.go

```diff
@@ -1723,15 +1723,15 @@ foo=bar
 				"hybrid overlay is not supported with ovnkube-node mode"))
 		})
 
-		It("Fails if management port is not provided and ovnkube node mode is dpu", func() {
+		It("Fails if management port is provided and ovnkube node mode is dpu", func() {
```

Comment:
Add test for DPU Host for this case:
`OvnKubeNode.Mode == types.NodeModeDPUHost && OvnKubeNode.MgmtPortNetdev == "" && OvnKubeNode.MgmtPortDPResourceName == ""`

Reply:
Can you clarify? We alredy have 

```
It("Fails if management port is not provided and ovnkube node mode is dpu-host", func() {
    cliConfig := config{
        OvnKubeNode: OvnKubeNodeConfig{
            Mode: types.NodeModeDPUHost,
        },
    }
    err := buildOvnKubeNodeConfig(nil, &cliConfig, &config{})
    gomega.Expect(err).To(gomega.HaveOccurred())
    gomega.Expect(err.Error()).To(gomega.ContainSubstring("ovnkube-node-mgmt-port-netdev or ovnkube-node-mgmt-port-dp-resource-name must be provided"))
})

```

Reply:
Ah sorry the test is below this one. Nevermind on this comment.